### PR TITLE
Change KafkaConsumer loop to consume all message sets

### PR DESCRIPTION
**Pull request by shanipribadi**
#79 pq should consume all MessageSets returned from poll

Change-Id: I232fa97a0136f32305f23b6510282288e5fcbd48

### CODE REVIEW

**Comment by sevagh** [comment]
Nice work!

I wonder why I ever put `thread::sleep(time::Duration::from_secs(1));` in the first place.

File: stream-delimit/src/kafka_consumer.rs [hunk 1]
```diff
@@ -1,13 +1,14 @@
 #![deny(missing_docs)]
 
 use kafka::consumer::{Consumer, FetchOffset};
-use std::{self, thread, time};
+use std::{self};
```
Comment (sevagh) [inline stream-delimit/src/kafka_consumer.rs]: `use std` is better

File: stream-delimit/src/kafka_consumer.rs [hunk 2]
```diff
@@ -18,30 +19,26 @@ impl Iterator for KafkaConsumer {
             let kafka_consumer = &mut self.consumer;
             loop {
                 match kafka_consumer.poll() {
-                    Ok(x) => match x.iter().take(1).next() {
-                        Some(y) => {
-                            self.messages.append(&mut y.messages()
-                                .iter()
-                                .map(|z| z.value.to_vec())
-                                .collect::<Vec<_>>());
+                    Ok(mss) => {
```
Comment (sevagh) [inline stream-delimit/src/kafka_consumer.rs]: The `rustfmt` formatting here has different indentation: 

```
                     Ok(mss) => {
                         for ms in mss.iter() {
                             self.messages.append(&mut ms.messages()
-                                                 .iter()
-                                                 .map(|z| z.value.to_vec())
-                                                 .collect::<VecDeque<_>>());
+                                .iter()
+                                .map(|z| z.value.to_vec())
+                                .collect::<VecDeque<_>>());
```

**Comment by shanipribadi** [comment]
@sevagh I have added the commit with your requested changes.

**Comment by sevagh** [comment]
Approved. Thanks for the contribution.

**Pull request merged**